NEWS.md

```diff
@@ -2,6 +2,8 @@
 
 ### Bug fixes
 
+* Fixed bug with spaces in path to the temporary folder on Windows. (#460)
```

Comment:
Lol that's a pretty good gotcha

Reply:
Yeah it was reported a few times, mostly with users that set their username as "firstname lastname".